check/src/docker.rs

```diff
@@ -37,15 +43,19 @@ fn create_image(version: &str) -> Result<()> {
         child.stdin.as_mut().unwrap(),
         "\
             FROM rust:{} as builder\n\
+            RUN rustup toolchain install {} && rustup default {}
             RUN rustup target add wasm32-unknown-unknown
             RUN rustup target add wasm32-wasi
             RUN rustup target add aarch64-unknown-linux-gnu
+            RUN rustup target add x86_64-unknown-linux-gnu
             RUN cargo install cargo-stylus
             RUN cargo install --force cargo-stylus-check
             RUN cargo install --force cargo-stylus-replay
             RUN cargo install --force cargo-stylus-cgen
         ",
-        version
+        version,
```

Comment:
Can you confirm that this version variable has also been cleaned to prevent injection attacks? @rauljordan

Reply:
Nice catch! Fixed